cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/UnmanagedClusterInfo/UnmanagedClusterInfo.tsx

```diff
@@ -0,0 +1,69 @@
+// React imports
+import React from 'react';
+
+// Library imports
+import { CdsButton } from '@cds/react/button';
+import { CdsIcon } from '@cds/react/icon';
+import { CdsDivider } from '@cds/react/divider';
+
+// App imports
+import './UnmanagedClusterInfo.scss';
+
+export interface UnmanagedClusterInfoProps {
+    name: string;
+    provider: string;
+    status: string;
+}
+
+function UnmanagedClusterInfo(props: Partial<UnmanagedClusterInfoProps>) {
```

Comment:
if we use `UnmanagedClusterInfoProps` instead of `Partial<UnmanagedClusterInfoProps>`, does it work?

Reply:
Yes it does, should I leave it as `UnmanagedClusterInfoProps`?

Reply:
yes. please use UnmanagedClusterInfoProps instead of Partial<UnmanagedClusterInfoProps>